hub/codec/image_codecs.py

```diff
@@ -0,0 +1,338 @@
+from io import BytesIO
```

Comment:
can we actually not call this package `hub.codec`? i would much prefer it be moved into core and instead of codec, call it `compression`. this path is more descriptive: `hub.core.compression`